site2/tools/build-site.sh

```diff
@@ -34,44 +34,55 @@ if [ "$CROWDIN_DOCUSAURUS_API_KEY" != "UNSET" ]; then
   # or current hour is 0-5
   # this leads to executing crowdin-upload once per day when website build is scheduled
   # to run with cron expression '0 */6 * * *'
-  CURRENT_HOUR=`date +%H`
+  CURRENT_HOUR=$(date +%H)
   if [[ "$CROWDIN_UPLOAD" == "1" || $CURRENT_HOUR -lt 6 ]]; then
     yarn run crowdin-upload
   fi
   yarn run crowdin-download
-fi
 
-# TODO: remove this after figuring out why crowdin removed code tab when generating translated files
```

Comment:
if you are touching this file, can you move all the `cp` commands into a shell function?

Reply:
Is that something @tuteng or you could take care of since I'm not familiar with Crowdin. This PR simply makes it possible to run and debug the site building locally without the Crowdin parts. 
I'd suggest that we do refactorings separately from this PR. Is that fine for you @sijie?

Reply:
there is nothing related to crowdin. this is just a review comment on the shell script.

Reply:
@sijie Are you expecting me to make some changes to this PR?

Reply:
ok, I'll extract a shell function.

Reply:
@sijie I have now extracted a shell function. PTAL

Reply:
I currently think this part can be removed, because newer versions since 2.5.2 have not added the corresponding content, but the site is built successfully, so I will consider doing a test to decide whether to remove the hack content